CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+# CONTRIBUTING
+
+Thank you for considering contributing to the Outrigger CLI!
+
+## Quality Contributions
+
+* Make sure your branch will compile.
+* Make sure your branch passes our static analysis checks.
+* Make sure your branch conforms with go fmt standards.
+* Manually test your changes.
+
+## User Interactions
```

Comment:
New CONTRIBUTING guide, which includes thoughts on logging that illustrates some of the API that's coming together.